app/assets/javascripts/select-kit/addon/components/future-date-input-selector.js

```diff
@@ -53,7 +53,7 @@ export const TIMEFRAMES = [
   buildTimeframe({
     id: "next_week",
     format: "ddd, h a",
-    enabled: (opts) => opts.day !== 7,
+    enabled: (opts) => opts.day !== 0,
```

Comment:
is this locale dependant by any chance?

Reply:
Nope, `day()` isn't locale-aware (there is another locale-aware function - [`weekday()`](https://momentjs.com/docs/#/get-set/weekday/)).

Reply:
More precisely, if you call `day()` using the name of weekday it'll be locale-aware. Day numbers aren't locale-aware.
```
moment().day(Number|String); // locale aware when passing String
moment().day(); // Number, isn't locale aware
```